streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedTableImpl.java

```diff
@@ -79,9 +79,12 @@
         // the passed in StreamsGraphNode must be the parent of the repartition node
         builder.addGraphNode(this.streamsGraphNode, repartitionNode);
 
-        final StatefulProcessorNode statefulProcessorNode = getStatefulProcessorNode(materialized,
-                                                                                     funcName,
-                                                                                     aggregateSupplier);
+        final StatefulProcessorNode statefulProcessorNode = new StatefulProcessorNode<K, Change<V>>(
+            funcName,
+            new ProcessorParameters<>(aggregateSupplier, funcName),
+            new KeyValueStoreMaterializer<>(materialized).materialize(),
+            false
+        );
```

Comment:
Having a separate method for `getStatefulProcessorNode` was unnecessary, so I've just inlined it to call the constructor directly instead. Perhaps that method did extra work in the past.